src/it/java/com/google/checkstyle/test/base/AbstractGoogleModuleTestSupport.java

```diff
@@ -81,6 +81,16 @@ protected DefaultConfiguration createModuleConfig(Class<?> clazz) {
         return new DefaultConfiguration(clazz.getName());
     }
 
+    /**
+     * Returns {@link Configuration} instance for {@link DefaultConfiguration} instance.
+     *
+     * @param moduleName  {@link DefaultConfiguration} instance.
+     * @return {@link Configuration} instance for {@link DefaultConfiguration} instance
+     */
+    protected static Configuration getModuleConfig(DefaultConfiguration moduleName) {
+        return moduleName;
```

Comment:
I did not get reason of this method.
as there are not overrides, looks likes can be removed.

Reply:
if checkstyle validation demands it, lets suppress violation in checkstyle config.

Reply:
Okay...I need a little time to recollect it why I did it and all. I will revert back to you soon